README.adoc

```diff
@@ -64,9 +64,9 @@ include::{common-includes}/gitclone.adoc[]
 
 include::{common-includes}/kube-start.adoc[]
 
-== Deploy the name and ping applications
+== Deploying the microservices
 
-You'll deploy two applications, `name` and `ping`. `name` displays a greeting and displays its container name. `ping` accepts a hostname as a parameter and returns `pong` if it can successfully make a request to `name`, otherwise the request fails. To build these applications, navigate to the `start` directory and run the following command.
+The two microservices you will deploy are called `name` and `ping`. The `name` microservice simply displays a brief greeting and the name of the container that it runs in. The `ping` microservice simply pings the {kube} Service that encapsulates the pod running the `name` microservice. This demonstrates how communication can be established between pods inside a cluster. To build these applications, navigate to the `start` directory and run the following command.
```

Comment:
The ping microservice simply pings the {kube} Service that...
=> I think the "simply" for the `ping` microservice is not needed
=> does "Service" need to be cap'ed here?

Reply:
I had the same question regarding Service but concluded that it should since we are referring to Kubernetes Service here.